nebullvm/config.py

```diff
@@ -43,5 +43,3 @@
     "tflite_model": "tf_model.tflite",
     "tf_model": "tf_model.h5",
 }
-
-SAVE_DIR_NAME = "optimized_model"
```

Comment:
Why are you reverting this? We don't want to save the model files into a directory anymore? Wouldn't this lead to an issue while saving large ONNX models?